tests/sample_decorator_appender_folder/test_decorator_appender_sample.py

```diff
@@ -0,0 +1,5 @@
+import pytest
+
+
+def test_pass():
```

Comment:
What does this file do? Is this a resource for testing?

Reply:
Is it possible to move this and similar into a `tests/resource` folder? This is the correct :tm: way of handling resources of this size that are used for testing purposes.

Reply:
Created the resource folder